smriprep/workflows/anatomical.py

```diff
@@ -1375,7 +1365,7 @@ def init_anat_template_wf(
     longitudinal: bool,
     omp_nthreads: int,
     num_files: int,
-    contrast: str,
+    image_type: ty.Literal['T1w', 'T2w'] = 'T1w',
```

Comment:
Does this need a default for some reason?

Reply:
I was intending to minimize changes in fmriprep

Reply:
I'd rather rip off the bandaid. Anything that you think should be generic, let's make it generic and deal with it now.

Reply:
all set - probably can squash all of this when merging